### PR TITLE
chore: fixed displaying provider names in Registry

### PR DESCRIPTION
**Pull request by tlakomy**

### CODE REVIEW

File: src/generator.ts [hunk 1]
```diff
@@ -153,6 +153,10 @@ export async function generateAll(
   }
 }
 
+function capitalizeFirstLetter(str: string) {
```
Comment (RafalWilinski) [inline src/generator.ts:156]: 🚢

**Review by RafalWilinski — APPROVED** [review]

File: schemas/ramp/v1/Card.json [hunk 2]
```diff
@@ -136,7 +136,7 @@
     "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
     "is_physical": true,
     "display_name": "string",
-    "last_four": "4049",
+    "last_four": "363",
```
Comment (asennikov) [inline schemas/ramp/v1/Card.json:139]: what's the reason for this change?

**Review by asennikov — COMMENTED** [review]
consider also updating the manually populated readme in `shopify/webhooks` folder: https://github.com/Stedi/registry/blob/main/schemas/shopify/webhooks/2022-01/README.md

File: schemas/ramp/v1/Card.json (hunk 2 above)
Comment (tlakomy) [reply schemas/ramp/v1/Card.json:139]: That's weird, I didn't touch this file, let me revert that.

**Review by asennikov — APPROVED** [review]
